tests/behat/features/capabilities/event-an-enroll/event-an-enroll.feature

```diff
@@ -0,0 +1,67 @@
+@api @javascript @event @eventenrollment @stability @perfect @PAC-423 @profile @stability-3 @event-an-enroll
+Feature: Enroll for an event without an account
+  Benefit: In order to attend an Event
+  Role: AN
+  Goal/desire: I want to be able to enroll to an event without having to create an account
+
+  @AN
+  Scenario: Successfully enroll to an event as AN user
```

Comment:
This test currently tests a lot more than the scenario suggests. Is it possible to split the code up into multiple scenario's so that it's clearer what is broken if the test fails?

Reply:
@Kingdutch I have fixed all items that you noticed, but would like to not split this behat test because of lack of time before upcoming release. Could you check again all changes and approve if you think it's ok?